.dependabot/config.yml

```diff
@@ -2014,3 +2014,31 @@ update_configs:
         dependency_name: demisto/*
         update_type: all
 
+  - package_manager: python
+    directory: /docker/reversinglabs-sdk-py3
+    update_schedule: live
+    automerged_updates:
+    - match:
+        update_type: semver:minor
+  - package_manager: docker
+    directory: /docker/reversinglabs-sdk-py3
+    update_schedule: daily
+    automerged_updates:
+    - match:
+        dependency_name: demisto/*
+        update_type: all
+
+  - package_manager: python
+    directory: /docker/reversinglabs-sdk-py2
```

Comment:
As we are not building py2 images, please remove the entries in this file regarding `reversinglabs-sdk-py2`

Reply:
did a sync with upstream and cleaned the py2 reference in .dependabot